crates/gpui/src/platform/linux/x11/client.rs

```diff
@@ -558,19 +565,19 @@ impl X11Client {
                 let window = self.get_window(event.window)?;
                 window.refresh();
             }
-            Event::FocusIn(event) => {
+            Event::FocusIn(event) if event.mode == xproto::NotifyMode::NORMAL => {
```

Comment:
I added these `if`s since the XinputLeave handler did the same, and from what I've read it seems like a good idea to do so.